pages/ActivityStreams/Overview.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Overview
+route: /ActivityStreams/Overview
+menu: Activity Streams
+---
+
+# Activity Streams Overview
+
+Content shared via the DSNP will generally consist of URIs pointing to documents containing Activity Streams JSON objects.
+For the purposes of the DSNP, only a subset of the [Activity Streams 2.0](https://www.w3.org/TR/activitystreams-core/) specification must be supported by default.
+Implementers may choose to support more of the Activity Streams standards as they see fit, but as a warning, these standards may include ambiguities which could lead to different and incompatible implementations of certain features.
+Implementers who extend their support for Activity Streams objects beyond the minimal subset defined here do so at their own risk.
+
+Additionally, some aspects of the Activity Streams standard may be modified or extended in the context of the DSNP to support specific validations or features necessary in a distributed context.
+These deviations from the standard will be thoroughly documented here.
+
+## Specification Status
+
+| Version | Status |
+---------- | ---------
+| 0.1     | Tentative |
+
+## Purpose
+
+1. Identify what aspects of the existing Activity Streams specification is supported, encouraged or enforced in the DSNP.
+1. Specify any extensions or modifications of the Activity Streams specification that must be supported or required by the DSNP.
+
+## Assumptions
+
+* All broadcast messages link to activity stream JSON, not any other form of content.
```

Comment:
I don't understand why this is an assumption. Why can't we assert it in the spec? If it's merely an assumption it's likely to be wrong. The header for this section could be "Links".

Reply:
That's a good point. These are not assumptions, they are important reminders of requirements elsewhere in the spec. Thoughts on how to title it? Links doesn't quite capture that to me, but it is better than Assumptions.

Reply:
This title seems just fine. I don't understand why the title makes it an assumption. It's called a "requirement" in the title.  Did this change after these comments were posted?  Did I miss something?  The only thing I'd change is  to put a "must" in both bullet points, i.e. "must link"